Documentation/ExtensionArchitecture/HowTo/ExtendExtbaseModel/Index.rst

```diff
@@ -0,0 +1,208 @@
+.. include:: /Includes.rst.txt
+
+.. _extending-extbase-model:
+
+==========================
+Extending an Extbase model
+==========================
+
+Once you have added an additional field to the database table and
+:ref:`TCA <extending-tca>` the new field will be displayed in the backend forms.
+
+However if the extension you are trying to extend is based on Extbase the new
+field is not available in the frontend out of the box. Further steps are
+needed to make the fields available. These steps will not work in all cases.
+
+Quick overview
+==============
+
+Follow these steps:
+
+..  rst-class:: bignums
+
+#.  Is your extension the :ref:`only one trying to
+    extend <extending-extbase-model_find_other_extending_models>` the original
+    model within your installation?
+
+#.  :ref:`Find the original model <extending-extbase-model_find_original_model>`.
+    If this model has the modifier :php:`final` refer
+    to the extensions documentation on how to display additional fields.
+
+#.  :ref:`Find the original repository <extending-extbase-model_find_original_repository>`.
+    Again, if the repository is :php:`final` refer
+    to the extensions documentation on how to display additional fields.
+
+#.  :ref:`Extend the original model <extending-extbase-model_extend_original_model>`
+    in your custom extension or sitepackage.
+
+#.  Register your extended model with the according database table in
+    :file:`EXT:my_extension/Configuration/Extbase/Persistence/Classes.php`.
+
+#.  :ref:`Extend the original repository <extending-extbase-model_register_extended_model>`
+    in your custom extension or sitepackage.
+
+#.  :ref:`Register your extended repository <extending-extbase-model_register_extended_repository>`
+    to be used instead of the original one.
+
+Step by step
+============
+
+..  _extending-extbase-model_find_other_extending_models:
+
+Are you the only one trying to extend that model?
+-------------------------------------------------
+
+Within your installation you can search for other classes that are extending
+the original model or repository (see bellow).
+
+If the model is already extended but you only need to create the fields for
+your current installation you can proceed by extending the extended model.
+
+If the model has different types you can decide to introduce a new type and
+only extend that one type. This is for example commonly done when extending
+the model of EXT:news.
```

Comment:
Please link EXT:news to the TER or the according documentation page. Not everyone may know that extension.